infra/cifuzz/actions/build_fuzzers/build_fuzzers_entrypoint.py

```diff
@@ -78,7 +81,7 @@ def main():
     logging.error('Error building fuzzers for project %s with pull request %s.',
                   oss_fuzz_project_name, pr_ref)
     return error_code
-  return 0
+  return not helper.check_build_impl(project_name, None)
```

Comment:
Isn't this supposed to be returning 0 on success, I think this will return True.

Reply:
Fixed.